examples/rust-examples/mips64/hello-world/.cargo/config.toml

```diff
@@ -0,0 +1,11 @@
+[build]
+rustflags = [
+    "-C", "link-arg=-Ttext=0x80010000",
+    "-C", "link-arg=-emain",
+]
+
+target = "mips64el-unknown-linux-gnuabi64"
+
+[unstable]
+build-std = ["core", "alloc"]
+
```

Comment:
Looks like you forgot
```
[target.mips64el-unknown-linux-gnuabi64]
linker = "lld"
```

Reply:
Initially i removed it because of "error: linker `lld` not found"